src/utils/global.css

```diff
@@ -27,6 +27,39 @@ body.dark {
   --inlineCode-text: #e6e6e6;
 }
 
+/*
+  RTL layout
+*/
+body:lang(fa) article {
```

Comment:
`:dir(rtl)` should work here.

Reply:
I don't think so. because `:dir(rtl)` is using for getting those elements which already set as RTL so we should set that first. And also it hasn't a [good browser support](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir) (available only on Firefox).

Reply:
I understand your point and agree with that. The solution is adding a class (like `rtl`) to body for detecting RTL languages but we can merge this PR and letting people to translate other posts then work on it in another PR or if it should be considered now let me know I will do it on this PR.